docs/configuration/snmp-data-format.md

```diff
@@ -117,12 +116,12 @@ The record that we'll see in Splunk `| mpreview index=net*` for the same case as
    metric_name:sc4snmp.IF-MIB.ifInUcastPkts: 47512921
 ```
 
-Note, that only fields specified in `varBinds` are actively polled form the device. In case of `profile_with_one_metric`
-shown above, the textual fields `ifAdminStatus`, `ifDescr`, `ifIndex`, `ifOperStatus` and `ifPhysAddress` are taken from 
-the database cache, which is updated on every `walk` process. This is fine for the most of the cases, as things like
-MAC address, interface type or interface status shouldn't change frequently if ever.
+Only fields specified in `varBinds` are actively polled from the device. In the case of the `profile_with_one_metric`
```

Comment:
Isn't there too much the in second sentence?

Reply:
How about this: "In the case of the previous`profile_with_one_metric` , the textual fields `ifAdminStatus`, `ifDescr`, `ifIndex`, `ifOperStatus` and `ifPhysAddress` are taken from the database cache. This is updated on every `walk` process."

Reply:
That one looks good.